playbooks/deployHIO.yml

```diff
@@ -94,3 +94,100 @@
     become: yes
     with_items:
       - "{{ groups['master'] }}"
+
+# Install Supervisord
+
+  - name: "install supervisord"
+    become: yes
+    apt:
+        name: "{{ item }}"
+        state: present
+        update_cache: yes
+    with_items:
+            - supervisor
+
+  - name: "supervisor service log directory"
+    become: yes
+    file:
+        path: /var/log/supervisor
+        state: directory
+
+  - name: "harmonicIO service log directory"
+    become: yes
+    file:
+        path: /home/ubuntu/HarmonicIO/logs
+        state: directory
+
+  - name: "start supervisord service"
+    become: yes
+    service:
+        name: supervisor
+        state: restarted
+
+- hosts: master
+
+  tasks:
+
+  - name: "setup supervisord service configuration file for harmonicIO master"
+    template:
+        src: ../configurationTemplates/harmonic_master.conf.j2
+        dest: /etc/supervisor/conf.d/harmonic_master.conf
+        owner: root
+    become: yes
+
+    # Supervisor cannot create log files if it doesn't exist
+  - name: "harmonicIO master service stdout stderr log files"
+    become: yes
+    file:
+        path: /home/ubuntu/HarmonicIO/logs/{{ item }}
+        state: touch
+    with_items:
+      - harmonic_master.log
+      - harmonic_master.err
+
+    # This task is necessary for supervisor to read child process configurations
+  - name: "add supervisord harmonicIO master service"
+    supervisorctl:
+        name: harmonic_master
+        state: present
+    become: yes
+
+  - name: "start supervisord harmonicIO master service"
+    supervisorctl:
+        name: harmonic_master
+        state: started
+    become: yes
+
+- hosts: workers
+
+  tasks:
+
+  - name: "setup supervisord service configuration file for harmonicIO worker"
+    template:
+        src: ../configurationTemplates/harmonic_worker.conf.j2
+        dest: /etc/supervisor/conf.d/harmonic_worker.conf
+        owner: root
+    become: yes
+
+    # Supervisor cannot create log files if it doesn't exist
+  - name: "harmonicIO worker service stdout stderr log files"
+    become: yes
+    file:
+        path: /home/ubuntu/HarmonicIO/logs/{{ item }}
```

Comment:
this directory is a clone of another repo. please make a PR to gitignore this directory (in HarmonicIO)